distr_tracing/distr_tracing_install/distr-tracing-deploying-otel.adoc

```diff
@@ -9,3 +9,7 @@ toc::[]
 The {OTELName} Operator uses a custom resource definition (CRD) file that defines the architecture and configuration settings to be used when creating and deploying the {OTELName} resources. You can either install the default configuration or modify the file to better suit your business requirements. 
 
 include::modules/distr-tracing-config-otel-collector.adoc[leveloffset=+1]
+
+include::modules/distr-tracing-config-otel-send-data-to-monitoring-stack.adoc[leveloffset=+1]
+Refer to xref:../../monitoring/enabling-monitoring-for-user-defined-projects.adoc[enable monitoring for user defined projects] how to
```

Comment:
Unless this is supposed to be part of the `distr-tracing-config-otel-send-data-to-monitoring-stack.adoc` module I'd add it to the intro paragraph or better yet to the Additional resources at the end of the assembly. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections

Reply:
fixed